_posts/2022-06-13-rails-discord-server-is-now-open-to-the-public.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Rails Discord server is now open to the public"
+categories: news
+author: rafaelfranca
+published: true
+date: 2022-06-13
+---
+
+In the past few weeks, the Rails team has been working on a few changes to lower the barrier for new contributors to the framework.
+
+We started by [disabling the automatic closing of stale PRs](https://github.com/rails/rails/commit/acf48169943011834c4c885b956e29050548fa98). This means that contributors can submit their PRs without
+the fear of a bot closing their worthy contributions before a member of the Rails Team has a chance to review them.
```

Comment:
Most usages of Rails team is without a capitalized team, so changing to be consistent:
```suggestion
the fear of a bot closing their worthy contributions before a member of the Rails team has a chance to review them.
```